### PR TITLE
added sps30 dockerfile & home sketch update for sps30

### PR DESCRIPTION
**Pull request by Thiemann96**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## v1.13.0
+
+## v.1.13.0
```
Comment (felixerdy) [inline CHANGELOG.md:7]: Can you remove this line `## v.1.13.0`?

**Review by felixerdy — CHANGES_REQUESTED** [review]
Just some minor things, otherwise looks good for me. You also want to add the sensor to the other templates?

File: templates/homev2_wifi.tpl [hunk 2]
```diff
@@ -28,6 +28,8 @@
 #include <LTR329.h>
 #include <ArduinoBearSSL.h>
 #include <Adafruit_DPS310.h> // http://librarymanager/All#Adafruit_DPS310
+#include <sps30.h> // http://librarymanager/All#
```
Comment (felixerdy) [inline templates/homev2_wifi.tpl:31]: If the library is not available in the arduino librarymanager, you can remove the comment

Comment (Thiemann96) [reply templates/homev2_wifi.tpl]: 9d6221a13a03013c85b6606629bb6fd5923c56eb

File: CHANGELOG.md (hunk 1 above)
Comment (Thiemann96) [reply CHANGELOG.md:5]: a7d132a5c1cef56e8569cf343be17d9ea5ceb100

**Pull request merged**